### DISCUSSION TITLE
تفاوت حالت جزفیلترنشده و فیلتر چیه؟

### DISCUSSION THREAD

**Discussion by hiddify-com** [hiddify/Hiddify-Manager#1076] [Content Help کمک به اصلاح محتوا]
قبل از هرچیزی بگیم نیاز داریم به پیشنهادات شما برای بهبود این بخش:
![image](https://github.com/hiddify/hiddify-config/assets/114227601/cbc56fc0-83af-4e2a-8326-d8292ff89a8f)

خوب اول بگیم چرا اصن دو تا لیست داریم؟؟؟
خوب میلیون ها وبسایت تو دنیا هست ما که نمیدونیم کدومشون فیلترن یا کدومشون نیستن!
پس اومدیم دو تا لیست درست کردیم. از اونایی که میدونیم فیلتر هستند و اونهایی که میدونیم فیلتر نیستند

# جز فیلتر نشده:

یعنی به جز سایت ها و اپلیکیشن هایی که میدونیم فیلتر نیستند، از فیلتر شکن عبور کنند.
مثال این لیست بانکهای ایران هست. 

[اگه اپلیکیشنی از قلم افتاده به این لیست اضافه کنید](https://github.com/hiddify/androidpackagenamelist/blob/master/direct_ir)

[اگه سایتی از قلم افتاده اینجا اضافه کنید](https://github.com/bootmortis/iran-hosted-domains)

# فیلتر شده:
یعنی فقط سایت ها و اپلیکیشن هایی که میدونیم فیلتر هستند از فیلتر شکن عبور کنند
یعنی به جز این سایت ها و اپلیکیشن ها همه بدون فیلترشکن وصل میشن
مثال این لیست تلگرام واتساپ و .... هست. 

[اگه اپلیکیشنی از قلم افتاده به این لیست اضافه کنید](https://github.com/hiddify/androidpackagenamelist/blob/master/proxy_ir)

اگه اپلیکیشنی از قلم افتاده به این لیست اضافه کنید
[اگه سایتی از قلم افتاده اینجا اضافه کنید](https://github.com/hiddify/domain-list-community/blob/master/data/gfw-ir)

**Comment by Ehsansajadieh** [comment]
حالت فیلتر شده کار نمیکنه و با حالت همه ، فرقی نداره

**Reply by ArefRahiimii** [reply]
برای اضافه یا حذف کردن برنامه ای چند لحظه ای انگشتتون رو نگه دارید روی لیست بعد براتون لیست برنامه ها باز میشه. مثلا روی فیلتر شده انگشتتون رو نگه دارید و برنامه های مورد علاقتون رو علامت بزنید تا با وی پی ان باز بشه

**Comment by Alijavidsohrab** [comment]
Good

**Comment by Alijavidsohrab** [comment]
I go to  shiraz

**Comment by skydarklord** [comment]
سلام و خسته نباشید
برای ویرایش فهرست سایتهای فیلتر شده و نشده باید از لیست شما یک فورک ایجاد کنیم ؟

**Reply by ArefRahiimii** [reply]
برای اضافه یا حذف کردن برنامه ای چند لحظه ای انگشتتون رو نگه دارید روی لیست بعد براتون لیست برنامه ها باز میشه. مثلا روی فیلتر شده انگشتتون رو نگه دارید و برنامه های مورد علاقتون رو علامت بزنید تا با وی پی ان باز بشه

**Reply by skydarklord** [reply]
سلام ممنون از راهنمایی شما . منظورم به فهرست سایتها و اپلیکیشنهای فلیتر شده و نشده است که دوستان در گیت هاب قرار دادند تا توسط سایرین ویرایش بشه .

**Comment by armagg** [comment]
اپ های تپسی و اسنپ رو از روی فیلتر شکن رد می‌کنه حالت فیلتر شده. این اوکیه؟

**Comment by 09961575820** [comment]
> قبل از هرچیزی بگیم نیاز داریم به پیشنهادات شما برای بهبود این بخش: ![image](https://private-user-images.githubusercontent.com/114227601/239677370-cbc56fc0-83af-4e2a-8326-d8292ff89a8f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MTE4MDMsIm5iZiI6MTcyMDQxMTUwMywicGF0aCI6Ii8xMTQyMjc2MDEvMjM5Njc3MzcwLWNiYzU2ZmMwLTgzYWYtNGUyYS04MzI2LWQ4MjkyZmY4OWE4Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwOFQwNDA1MDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hZTgzMTQ0NGQyYjAwYmZhY2M1NjFkODE4ZjFmMjMxZDU3ZjZjYWYzODk2MTcxMDZmODNjNGY3ZTJhYTJhOWMxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.-Sg6fi3RF_dxY4GBtSu0Ww82llQ118CFCZguzNTH-Lc)
> 
> خوب اول بگیم چرا اصن دو تا لیست داریم؟؟؟ خوب میلیون ها وبسایت تو دنیا هست ما که نمیدونیم کدومشون فیلترن یا کدومشون نیستن! پس اومدیم دو تا لیست درست کردیم. از اونایی که میدونیم فیلتر هستند و اونهایی که میدونیم فیلتر نیستند
> 
> # جز فیلتر نشده:
> یعنی به جز سایت ها و اپلیکیشن هایی که میدونیم فیلتر نیستند، از فیلتر شکن عبور کنند. مثال این لیست بانکهای ایران هست.
> 
> [اگه اپلیکیشنی از قلم افتاده به این لیست اضافه کنید](https://github.com/hiddify/androidpackagenamelist/blob/master/direct_ir)
> 
> [اگه سایتی از قلم افتاده اینجا اضافه کنید](https://github.com/bootmortis/iran-hosted-domains)
> 
> # فیلتر شده:
> یعنی فقط سایت ها و اپلیکیشن هایی که میدونیم فیلتر هستند از فیلتر شکن عبور کنند یعنی به جز این سایت ها و اپلیکیشن ها همه بدون فیلترشکن وصل میشن مثال این لیست تلگرام واتساپ و .... هست.
> 
> [اگه اپلیکیشنی از قلم افتاده به این لیست اضافه کنید](https://github.com/hiddify/androidpackagenamelist/blob/master/proxy_ir)
> 
> اگه اپلیکیشنی از قلم افتاده به این لیست اضافه کنید [اگه سایتی از قلم افتاده اینجا اضافه کنید](https://github.com/hiddify/domain-list-community/blob/master/data/gfw-ir)

- [ ]  #[](url)****

**Comment by 09961575820** [comment]
``
